src/forge/losses/grpo_loss.py

```diff
@@ -7,23 +7,26 @@
 import torch
 from torch import nn
 
+from forge.data_models.loss_metrics import LossMetrics
```

Comment:
I'm not sure this is a fully fleshed out data model we want to use. 

For now could we just define a loose type in this file and shove the metrics in that?